.github/workflows/codeql-analysis.yml

```diff
@@ -53,7 +53,7 @@ jobs:
     # Autobuild attempts to build any compiled languages  (C/C++, C#, or Java).
     # If this step fails, then you should remove it and run the build manually (see below)
     - name: Autobuild
-      uses: github/codeql-action/autobuild@v1
+      uses: github/codeql-action/autobuild@v2
```

Comment:
I don't think we need to run autobuild right? As our workflow does that

Reply:
Honestly that part kind of confuses me. Looking at some issues https://github.com/github/codeql-action/issues/365

I wonder if it's analyzing the typescript directly, if it should be dealing with the built javascript and map files.

Reply:
It supports TS so I don't think so? But I dunno tbh.